### ISSUE TITLE
Potential wrong selection to compose combined_obs for AMP discriminator

### ISSUE THREAD

**Issue by zhuoqun-chen** [NVlabs/ProtoMotions#125]
In current implementation AMP agent implementation, the `combined_obs` logic combines current agent observation data and expert observation data.

https://github.com/NVlabs/ProtoMotions/blob/9f8caaf114279f15ee4d29eb04fb9f9a754c50ea/protomotions/agents/amp/agent.py#L195

Howoever, I think the correct logic should be combining only agent its own data, which are current agent observation data and the history replay buffer's observation, which is https://github.com/nv-tlabs/ASE/blob/21257078f0c6bf75ee4f02626260d7cf2c48fee0/ase/learning/amp_agent.py#L365:

```python
disc_agent_cat_logit = torch.cat([disc_agent_logit, disc_agent_replay_logit], dim=0)
```

**Comment by zhuoqun-chen** [comment]
oops, I think I made a mistake. It seems that the replay obs is processed to logits separately. please just ignore this issue.
